docs/user/filesystem.md

```diff
@@ -799,6 +799,15 @@ $ LD_PRELOAD=/usr/lib64/libioil.so dd if=/dev/zero of=./bar bs=1G count=20
 Alternatively, it's possible to simply link the interception library into the application
 at compile time with the `-lioil` flag.
 
+By default, daos is only initialized in the interception library if the application calls
```

Comment:
It's not if/when it calls open, it's when it detects the first file descriptor backed by dfuse. It does the ioctl after every open and then calls daos_init only if it gets a reply from the ioctl.